app/assets/stylesheets/service_calendar.scss

```diff
@@ -120,6 +121,10 @@
   }
 }
 
+#select-t-qty {
```

Comment:
Instead of specifying the margin here, you can remove the `mb-0` from the 3 quantities `form-group`s which will then have a built-in `margin-bottom` for spacing with the stuff you added